rust/datafusion/src/optimizer/filter_push_down.rs

```diff
@@ -126,13 +126,13 @@ impl OptimizerRule for FilterPushDown {
             }
 
             // AND filter expressions that would be placed on the same depth
-            if let Some(existing_expression) = new_filtersnew_filters.get(&new_depth) {
```

Comment:
I am not sure what was going on with this variable name, but I figured I would reduce the replication ;)